### PR TITLE
Generate release SQL patch and add missing entries in the CHANGELOG for 24.1 release

### PR DESCRIPTION
**Pull request by cmadjar**
## Brief summary of changes

Generate the release SQL patch, move the patches to Archive and add missing entries in the CHANGELOG for 24.1 release.

### CODE REVIEW

File: CHANGELOG.md
Comment (driusan) [inline CHANGELOG.md:22]:
```suggestion
- Ability to use PSCID instead of the CandID in the candidates API (PR #8138)
```

File: CHANGELOG.md
Comment (driusan) [inline CHANGELOG.md:28]: I don't think the above 3 need to be mentioned because I think they were just broken on the branch and not the latest release, but "Add support for PHP 8.1" should probably be mentioned

Comment (cmadjar) [reply CHANGELOG.md]: @driusan removed!

Comment (cmadjar) [reply CHANGELOG.md]: oops. missed one part of your comment. Added PHP 8.1 support in last commit.

**Review by driusan — CHANGES_REQUESTED** [review]
Just noticed it's missing the "## LORIS 24.1 (Release Date: ???)" header and the "Support for PHP 8.1" should probably go under Core instead of modules.. other than that looks good but I'll wait until the other PRs are merged before merging this one in case something else comes up and so we can put the date right before releasing.

**Comment by cmadjar** [comment]
@driusan done. Let me know if you see anything else

File: CHANGELOG.md
Comment (cmadjar) [inline CHANGELOG.md:11]:
```suggestion
## LORIS 24.1 (Release Date: 2022-10-04)
```

**Pull request merged**
